zone/lua_mod.cpp

```diff
@@ -678,4 +679,74 @@ void LuaMod::CalcSpellEffectValue_formula(Mob *self, uint32 formula, int64 base_
 	}
 }
 
+
+void LuaMod::RegisterBug(Client *self, BaseBugReportsRepository::BugReports bug, bool &ignoreDefault)
+{
+	int start = lua_gettop(L);
+
+	try {
+		if (!m_has_register_bug) {
+			return;
+		}
+
+		lua_getfield(L, LUA_REGISTRYINDEX, package_name_.c_str());
+		lua_getfield(L, -1, "RegisterBug");
+
+		Lua_Client l_self(self);
+		luabind::adl::object e = luabind::newtable(L);
+		e["self"] = l_self;
+		e["zone"] = bug.zone;
+		e["client_version_id"] = bug.client_version_id;
+		e["client_version_name"] = bug.client_version_name;
+		e["account_id"] = bug.account_id;
+		e["character_id"] = bug.character_id;
+		e["character_name"] = bug.character_name;
+		e["reporter_spoof"] = bug.reporter_spoof;
+		e["category_id"] = bug.category_id;
+		e["category_name"] = bug.category_name;
+		e["reporter_name"] = bug.reporter_name;
+		e["ui_path"] = bug.ui_path;
+		e["pos_x"] = bug.pos_x;
+		e["pos_y"] = bug.pos_y;
+		e["pos_z"] = bug.pos_z;
+		e["heading"] = bug.heading;
+		e["time_played"] = bug.time_played;
+		e["target_id"] = bug.target_id;
+		e["target_name"] = bug.target_name;
+		e["optional_info_mask"] = bug.optional_info_mask;
+		e["_can_duplicate"] = bug._can_duplicate;
+		e["_crash_bug"] = bug._crash_bug;
+		e["_target_info"] = bug._target_info;
+		e["_character_flags"] = bug._character_flags;
+		e["_unknown_value"] = bug._unknown_value;
+		e["bug_report"] = bug.bug_report;
+		e["system_info"] = bug.system_info;
```

Comment:
```cpp
e["self"]                = l_self;
e["zone"]                = bug.zone;
e["client_version_id"]   = bug.client_version_id;
e["client_version_name"] = bug.client_version_name;
e["account_id"]          = bug.account_id;
e["character_id"]        = bug.character_id;
e["character_name"]      = bug.character_name;
e["reporter_spoof"]      = bug.reporter_spoof;
e["category_id"]         = bug.category_id;
e["category_name"]       = bug.category_name;
e["reporter_name"]       = bug.reporter_name;
e["ui_path"]             = bug.ui_path;
e["pos_x"]               = bug.pos_x;
e["pos_y"]               = bug.pos_y;
e["pos_z"]               = bug.pos_z;
e["heading"]             = bug.heading;
e["time_played"]         = bug.time_played;
e["target_id"]           = bug.target_id;
e["target_name"]         = bug.target_name;
e["optional_info_mask"]  = bug.optional_info_mask;
e["_can_duplicate"]      = bug._can_duplicate;
e["_crash_bug"]          = bug._crash_bug;
e["_target_info"]        = bug._target_info;
e["_character_flags"]    = bug._character_flags;
e["_unknown_value"]      = bug._unknown_value;
e["bug_report"]          = bug.bug_report;
e["system_info"]         = bug.system_info;
```

Reply:
Is this a proposed change? I can't tell.. maybe tab fixing?

Reply:
Just a formatting thing.